Src/DependencyCollector/Shared/DependencyTrackingTelemetryModule.cs

```diff
@@ -237,7 +242,7 @@ private void InitializeForDiagnosticAndFrameworkEventSource()
                     this.SetComponentCorrelationHttpHeaders,
                     this.ExcludeComponentCorrelationHttpHeadersOnDomains,
                     this.EffectiveProfileQueryEndpoint);
-                this.httpDesktopDiagnosticSourceListener = new HttpDesktopDiagnosticSourceListener(desktopHttpProcessing);
+                this.httpDesktopDiagnosticSourceListener = new HttpDesktopDiagnosticSourceListener(desktopHttpProcessing, new ApplicationInsightsUrlFilter(this.telemetryConfiguration));
```

Comment:
Minor: Does it mean that the only moment we assign telemetry configuration to Url filter is the initialize of the dependency tracking? Change to the telemetry channel endpoint after that, e.g. in the code will not be part of the Url filter and endpoint set in code might not be validated in the filter. It might've been like that before, though.